test/browser/index.js

```diff
@@ -23,25 +23,19 @@ describe('App', () => {
 
   describe('routing', () => {
```

Comment:
I'm not really a fan of commenting out tests...

I would definitely vote for creating an issue, with the test source code included in the ticket description, so it can be pulled out of the actual source code entirely.

Commented out code has a nasty habit of lingering way longer than anyone intends for it to